btf/btf.go

```diff
@@ -824,7 +820,7 @@ var haveBTF = internal.FeatureTest("BTF", "5.1", func() error {
 })
 
 var haveFuncLinkage = internal.FeatureTest("BTF func linkage", "5.6", func() error {
-	if err := haveBTF(); err != nil {
+	if err := haveMapBTF(); err != nil {
```

Comment:
Seems like a weird callchain.

Reply:
I think you posted this comment on the particular commit where `haveBTF()` was renamed to `haveMapBTF()`. A subsequent commit changes this to `haveProgBTF()`.